src/data/markdown/docs/40 xk6-disruptor/01 Get started/02 Requirements.md

```diff
@@ -0,0 +1,22 @@
+---
+title: 'Requirements'
+excerpt: 'Requirements for using xk6-disruptor in your test scripts'
+---
+
+The `xk6-disruptor` is a `k6` extension. In order to use it in a `k6` test script, it is necessary to use a custom build of `k6` that includes the disruptor. See the [Installation](/javascript-api/xk6-disruptor/get-started/installation) section  for instructions on how to get this custom build.
+
+The `xk6-disruptor` needs to interact with the Kubernetes cluster on which the application under test is running. In order to do so, you must have the credentials to access the cluster in a [kubeconfig](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) file. Ensure this file is pointed to by the `KUBECONFIG` environment variable or it is located at the default location `$HOME/.kube/config`.
+
+<Blockquote mod="attention">
```

Comment:
I think this is an important note that should b highlighted. I prefer to keep attention here.

Reply:
So, I just looked it up, and according the Google style guide, version requirements shouldn't be in notes or cautions. So, my recommendation and yours were against guidelines :handshake: :confused:  

Instead, let's put the sentence in bold at the end of the first paragraph.

https://developers.google.com/style/notices

Reply:
I also wonder whether it's completely correct: does xk6 require this kubernetes version, or is only compatible with this version.

Require would imply it won't work without it. Compatible would imply it can only interact with this version or later.

Reply:
Requires: it won't work with previous versions. I will change for note but I'm not convinced because if user uses the disruptor with previous versions it will get error messages.